packages/api-gateway/src/controller/account/account.controller.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Patch,
+  Body,
+} from '@nestjs/common';
+import { CreateAccountDto, ResponseAccountDto, UpdateAccountDto } from './dto';
+import {
+  ApiBody,
+  ApiOkResponse,
+  ApiOperation,
+  ApiParam,
+  ApiTags,
+} from '@nestjs/swagger';
+
+@ApiTags('Account API')
+@Controller('account')
+export class AccountController {
+  @ApiOperation({
+    summary: 'Returns account',
+    description: 'Returns account with the same UUID',
+  })
+  @ApiParam({
+    name: 'id',
+    description: 'the string representation of the target account UUID',
+  })
+  @ApiOkResponse({ type: ResponseAccountDto })
+  @Get(':id')
```

Comment:
this must be first, e.g.
@Get(':id')
@ApiOperation({
    summary: 'Returns account',
    description: 'Returns account with the same UUID',
  })
    @ApiParam({
    name: 'id',
    description: 'the string representation of the target account UUID',
  })
  @ApiOkResponse({ type: ResponseAccountDto })

Reply:
+